product_docs/docs/pem/8/pem_query_tool.mdx

```diff
@@ -238,26 +238,26 @@ Use the `Connection status` feature to view the current connection and transacti
 
 ## Macros
 
-Query Tool Macros enable you to execute pre-defined SQL queries with a single key press. Pre-defined queries can contain the placeholder $SELECTION$. Upon macro execution, the placeholder will be replaced with the currently selected text in the Query Editor pane of the Query Tool.
+Query Tool Macros enable you to execute pre-defined SQL queries with a single key-press. Pre-defined queries can contain the placeholder $SELECTION$. Upon macro execution, the placeholder will be replaced with the currently selected text in the Query Editor pane of the Query Tool.
 
 ![Query Tool - Connection Status button](images/query_tool_manage_macros.png)
 
-To create a macro, select the *Manage Macros* option from the *Macros* menu on the *Query Tool*. Select the key you wish to use, enter the name of the macro, and the query, optionally including the selection placeholder, and then click the *Save* button to store the macro.
+To create a macro, select the `Manage Macros` option from the Macros menu on the Query Tool. Select the key you wish to use, enter the name of the macro, and the query, optionally including the selection placeholder, and then click the `Save` button to store the macro.
 
 ![Query Tool - Connection Status button](images/query_tool_manage_macros_dialog.png)
 
-To clear a macro, select the macro on the *Manage Macros* dialogue, and then click the *Clear* button.
+To clear a macro, select the macro on the `Manage Macros` dialogue, and then click the `Clear` button.
 
 ![Query Tool - Connection Status button](images/query_tool_macros_clear_row.png)
 
-The server will prompt you for confirmation to clear the macro.
+In the sever prompt, click `Yes` to clear the selected macro. 
 
 ![Query Tool - Connection Status button](images/query_tool_macros_clear_confirmation.png)
 
-To clear all macros, click on the *Clear* button on left side of the key. The server will prompt you for confirmation to clear all the rows.
+To clear all macros, click on the `Clear` button on left side of the key. In the server prompt, click `Yes` to clear all the rows.
```

Comment:
Select vs click on, no "button" Same issue about the prompt:

To clear all macros, select **Clear** on left side of the key. To clear all rows, respond **Yes** to the prompt.